cache

```diff
@@ -71,8 +223,9 @@ cache::store() {
   usage='Usage: \n\t cache store [key] [local-path]\nExample: \n\t cache store bundle-$SEMAPHORE_GIT_BRANCH-$(checksum Gemfile.lock) vendor/bundle\n'
```

Comment:
🙏 pls consider describing the `cache store` usage by adding it to the usage msg. With the introduced changes, when `cache store` is being called with wrong number of arguments, its output log doesn't mention `cache store`.

Example:
<img width="1281" alt="image" src="https://user-images.githubusercontent.com/10695735/61110645-ff2a1900-a487-11e9-9267-ddbbf52a2700.png">